pyccel/ast/builtin_methods/dict_methods.py

```diff
@@ -60,6 +60,16 @@ def dict_obj(self):
         """
         return self._dict_obj
 
+    @property
+    def modified_args(self):
+        """
+        Return an iterable of all the arguments which may be modified by this function.
+
+        Return an iterable of all the arguments which may be modified by this function.
+        This is notably useful in order to determine the constness of arguments.
+        """
+        return (self._dict_obj,)
```

Comment:
Does the fact that this function returns `self._dict_obj` mean that in general the dict methods modify the dict itself?

Reply:
Yes exactly

Reply:
All clear then, thanks!